client/src/components/landing/index.js

```diff
@@ -16,14 +23,41 @@ const propTypes = {
 
 const BigCallToAction = () => (
   <Row>
-    <Col md={6} mdOffset={3} sm={10} smOffset={1} xs={12}>
+    <Col md={8} mdOffset={2} sm={10} smOffset={1} xs={12}>
       <Login block={true} data-test-label='landing-big-cta'>
-        Sign in and get started (it's free)
+        Get started (it's free)
       </Login>
     </Col>
   </Row>
 );
 
+const CompanyLogosSection = () => (
+  <div className='logo-row'>
+    <Col md={2} sm={3} xs={4}>
+      <AppleLogo />
+    </Col>
+    <Col md={2} sm={3} xs={4}>
+      <GoogleLogo />
+    </Col>
+    <Col md={2} sm={3} xs={4}>
+      <AmazonLogo />
+    </Col>
+    <Col md={2} sm={3} xs={4}>
+      <MicrosoftLogo />
+    </Col>
+    <Col md={2} sm={3} xs={4}>
+      <SpotifyLogo />
+    </Col>
+  </div>
+);
+
+const AsFeaturedSection = () => (
+  <div className='text-center'>
+    <h2 className='medium-heading'>As Featured In:</h2>
+    <AsFeatureLogo width='70%' />
+  </div>
+);
+
```

Comment:
@ahmadabdolsaheb should this/these be separate components?

Reply:
good point. I think we can remove the bootstrap and style it with flexbox to have more control over the breakpoints and keep it short

Reply:
ah okay, I see that you fixed it in clean up commit right? @ahmadabdolsaheb

Reply:
@thecodingaviator, I placed the company logos into a separate component.

Next step would be to merge the upper part of landing and into components together since they are almost identical and render the difference conditionally.

Reply:
@ahmadabdolsaheb You want to merge [this](https://github.com/freeCodeCamp/freeCodeCamp/blob/86e450d1a4c903a33d383cd006157eb13ec61b1e/client/src/components/landing/index.js#L78) and [this](https://github.com/freeCodeCamp/freeCodeCamp/blob/86e450d1a4c903a33d383cd006157eb13ec61b1e/client/src/components/landing/index.js#L52)?

Reply:
To clarify, we are trying to make this an individual component that we could use on the intro component, and landing component.

![image](https://user-images.githubusercontent.com/4591597/81164095-74775500-8f98-11ea-97c7-feac6c2d9cf3.png)

The featured in section can stay as is.

Reply:
Do you want to take that up on another PR maybe?